README.md

```diff
@@ -44,6 +44,26 @@ let make = (~name, _children) => {
 };
 ```
 
+If you want to avoid repeating the package name you can use the context namespacing:
+
+```reason
+let component = ReasonReact.statelessComponent("Header");
+
+let make = (~name, _children) => {
+  ...component,
+  render: (_self) =>
+    MaterialUI.(
+      <AppBar position="static" color="default">
+        <Toolbar>
+          <Typography _type="title" color="inherit">
+            (ReasonReact.stringToElement("Title"))
+          </Typography>
+        </Toolbar>
+      </AppBar>
+    )
+};
+```
+
```

Comment:
Good idea :+1:  

We should probably just replace the other example with this one, we don't need both or the explanation.